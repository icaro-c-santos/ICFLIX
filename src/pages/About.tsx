import { Container, Typography } from "@mui/material";
import { Link } from "react-router-dom";

export const About = () => {
  return (
    <>
      <Container
        maxWidth="md"
        sx={{
          bgcolor: "#bfa38d",
          borderRadius: "25px",
          textAlign: "justify",
          marginTop:"100px"
        }}
      >
        <Typography sx={{ fontSize: "25px", padding: "30px 40px", }}>
          <p>
            Este site foi produzido por Ícaro Santos, com o intuito de aprender
            sobre o uso dos componente do Material UI e também para estudar
            alguns conceitos do React, dentre eles: react-dom, react-routers,
            react-context entre outros.
          </p>
          <p>
            O intutito desse site é apenas o front-end da aplicação, foi criado
            um back-end fake para servir alguns dados de autenticação para a
            aplicação e para servir os dados dos filmes foi utilizado a api do
            themoviedb "https://www.themoviedb.org/documentation/api".
          </p>
        </Typography>
      </Container>
    </>
  );
};
