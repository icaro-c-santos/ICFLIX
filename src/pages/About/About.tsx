import { Container, Typography } from "@mui/material";
import Box from "@mui/material/Box";
import { Link } from "react-router-dom";

export const About = () => {
  return (
    <Box sx={{minHeight:"484.5px"}}>
    <Container
        maxWidth="md"
        sx={{
          bgcolor: "#bfa38d",
          borderRadius: "25px",
          textAlign: "justify",
          marginTop:"100px",
        }}
      >
        <Typography sx={{ fontSize: "25px", padding: "30px 40px", }}>
          <span>
            Este site foi produzido por Ícaro Santos, com o intuito de aprender
            sobre o uso dos componente do Material UI e também para estudar
            alguns conceitos do React, dentre eles: react-dom, react-routers,
            react-context entre outros.
          </span>
          <span>
            O intutito desse site é apenas o front-end da aplicação, foi criado
            um back-end fake para servir alguns dados de autenticação para a
            aplicação e para servir os dados  foi utilizado a api do
            rick and morty "https://rickandmortyapi.com/documentation/".
          </span>
        </Typography>
      </Container>
    </Box>
  );
};
