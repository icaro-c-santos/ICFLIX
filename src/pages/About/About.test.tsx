import { About } from "./About";
import renderer from "react-test-renderer";
it("Matches DOM Snapshot", () => {
  const domTree = renderer.create(<About></About>).toJSON();
  expect(domTree).toMatchSnapshot();
});
