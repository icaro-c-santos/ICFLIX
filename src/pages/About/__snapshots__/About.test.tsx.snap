// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches DOM Snapshot 1`] = `
<div
  className="MuiContainer-root MuiContainer-maxWidthMd css-kwaj63-MuiContainer-root"
>
  <p
    className="MuiTypography-root MuiTypography-body1 css-a04nev-MuiTypography-root"
  >
    <span>
      Este site foi produzido por Ícaro Santos, com o intuito de aprender sobre o uso dos componente do Material UI e também para estudar alguns conceitos do React, dentre eles: react-dom, react-routers, react-context entre outros.
    </span>
    <span>
      O intutito desse site é apenas o front-end da aplicação, foi criado um back-end fake para servir alguns dados de autenticação para a aplicação e para servir os dados dos filmes foi utilizado a api do themoviedb "https://www.themoviedb.org/documentation/api".
    </span>
  </p>
</div>
`;
